test/plugin/scenarios/netty-socketio-scenario/config/expectedData.yaml

```diff
@@ -0,0 +1,97 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+registryItems:
+  applications:
+    - {netty-socketio-scenario: 2}
+  instances:
+    - {netty-socketio-scenario: 1}
+  operationNames:
+    - netty-socketio-scenario: [/netty-socketio-scenario/case/netty-socketio, /socket.io/,
+                                /netty-socketio-scenario/healthCheck, SocketIO/onConnect, SocketIO/send_data/receive]
+  heartbeat: []
+segmentItems:
+  - applicationCode: netty-socketio-scenario
+    segmentSize: ge 5
```

Comment:
Why is ```ge 5```, but only four ```segments```,  ```/case/netty-socketio -local- >  io.socket.client.Socket -CrossProcess -> SocketIOServer ``` and ```/case/netty-socketio -local- > SocketIOClient  -CrossProcess-> io.socket.client.Socket```  whether this result is more reasonable? just for reference

Reply:
cause In this plugin, we havent provide socket.io-client intercept.